src/bin/nydusd/fusedev.rs

```diff
@@ -97,10 +97,10 @@ struct FuseServer {
 }
 
 impl FuseServer {
-    fn new(server: Arc<Server<Arc<Vfs>>>, se: &FuseSession, evtfd: EventFd) -> Result<FuseServer> {
+    fn new(server: Arc<Server<Arc<Vfs>>>, se: &FuseSession, _evtfd: EventFd) -> Result<FuseServer> {
         Ok(FuseServer {
             server,
-            ch: se.new_channel(evtfd)?,
+            ch: se.new_channel()?,
```

Comment:
I thought the patch is to do a plain replacement of `linux_abi`, but this seems irrelevant?

Reply:
The `new_channel` function signature has also changed.